File: Praktikum 1. Variable, Tipe Data, dan Operator/Bahasa C/PRAK110-2210817110014-Muhammad_Firas.c
#include<stdio.h>
#include<math.h>

int main()
{
    double alas = 5;
    double tinggi = 12;

    printf("Diketahui : \n");
    printf("Alas = %1.0lf cm\n", alas);
    printf("Tinggi = %1.0lf cm\n \n", tinggi);
    printf("Jawab : \n");
    printf("Sisi A = %1.0lf cm\n", tinggi);
    //selanjutnya kita gunakan rumus phytagoras supaya dapat sisi B
    printf("Sisi B = %1.0lf cm\n", sqrt((alas*alas)+(tinggi*tinggi)));
    double miring = 13;
    printf("Sisi C = %1.0lf cm\n", alas);
    printf("Keliling = %1.0lf cm\n", alas+tinggi+miring);
    //lalu pakai rumus luas segitiga
    printf("Luas = %1.0lf cm", 0.5*alas*tinggi);
}

File: Praktikum 1. Variable, Tipe Data, dan Operator/Bahasa Python/PRAK110-2210817110014-Muhammad Firas.py
import math
alas = 5
tinggi = 12

print("Diketahui :")
print("Alas = %d cm" %alas)
print("Tinggi = %d cm \n" %tinggi)
print("Jawab :")
print("Sisi A = %d cm" %tinggi)
#menggunakan rumus phytagoras
print("Sisi B = %d cm" %round(math.sqrt((alas**2)+(tinggi**2))))
miring = 13
print("Sisi C = %d cm" %alas)
print("Keliling = %d cm" %(alas+tinggi+miring))
print("Luas = %d cm" %(1/2*alas*tinggi))